core/src/test/java/org/infinispan/filter/CompositeKeyValueFilterConverter.java

```diff
@@ -14,6 +19,7 @@
  * @author wburns
  * @since 7.0
  */
+@SerializeWith(CompositeKeyValueFilterConverter.Ext.class)
```

Comment:
In fact, this change is not required, or at least not in its current form.

I originally put it in when trying to make sure no Infinispan classes were being serialized with the assumption that they extended Serializable. Hence I added the annotation based externalizer for this test class. However, once #4627 is in, it will probably just extend `ExternalPojo` and it won't need the annotation.

So, I'll go ahead and remove this change.